src/js/components/DataTable/ExpanderCell.js

```diff
@@ -38,7 +45,7 @@ const ExpanderControl = ({ context, expanded, onToggle, pad, ...rest }) => {
     content = (
       <Button
         fill
-        a11yTitle={expanded ? 'collapse' : 'expand'}
+        a11yTitle={`${expanded ? 'collapse' : 'expand'} ${primaryValue}`}
```

Comment:
This may not be human friendly (really in most cases it should be non visible id or guid)